src/domain/gift-cards/details/index.tsx

```diff
@@ -25,12 +26,12 @@ import UpdateBalanceModal from "./update-balance-modal"
 const GiftCardDetails: React.FC<RouteComponentProps<{ id: string }>> = ({
   id,
 }) => {
-  const { gift_card, isLoading } = useAdminGiftCard(id!, {
+  const { gift_card: giftCard, isLoading } = useAdminGiftCard(id!, {
     enabled: !!id,
   })
   const { regions } = useAdminRegions()
 
-  const updateGiftCard = useAdminUpdateGiftCard(gift_card?.id!)
+  const updateGiftCard = useAdminUpdateGiftCard(giftCard?.id!)
```

Comment:
Would cause an error if gift_card is not load loaded yet and we then try to access id. Doing it like giftCard?.id! makes sense as we avoid errors when the page is first loaded, but at the time where updateGiftCard is actually invoked we know that the gift card is loaded.